.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,5 @@
+- id: helm-unittest
+  name: helm-unittest
+  language: docker_image
+  entry: helmunittest/helm-unittest:3.17.1-0.8.0
```

Comment:
Is there a build script / process that automatically updates version references?